.github/workflows/build-and-push.yml

```diff
@@ -60,7 +60,8 @@ jobs:
       - name: Build and push runtime image
         uses: docker/build-push-action@v4
         with:
-          platforms: linux/amd64,linux/arm64,linux/ppc64le,linux/s390x
+          # tensorflow not supported for s390x architecture https://github.com/tensorflow/tensorflow/issues/46181
+          platforms: linux/amd64,linux/arm64,linux/ppc64le #,linux/s390x
```

Comment:
same for ppc64le ;( https://pypi.org/project/tensorflow/2.12.0/#files

Reply:
Right, thank you, I just noticed your comment. Only `linux/amd64` and `linux/arm64` available

- tensorflow-2.12.0-cp311-cp311-**win_amd64**.whl
- tensorflow-2.12.0-cp311-cp311-manylinux_2_17_x86_64.many**linux**2014_**x86_64**.whl (`linux/amd64`)
- tensorflow-2.12.0-cp311-cp311-manylinux_2_17_aarch64.many**linux**2014_**aarch64**.whl (`linux/arm64`)
- tensorflow-2.12.0-cp311-cp311-macosx_10_15_x86_64.whl